package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "mn-bootstrap",
+  "version": "0.12.0-dev.1",
```

Comment:
0.13.0-dev.1

Reply:
Shouldn't this be a tool for 0.12.x though?

Reply:
12 is released and considering as stable (master branch) so just fixes should go there but new untested features should go to dev branch.